docs/bokeh/source/docs/first_steps/first_steps_1.rst

```diff
@@ -54,21 +60,24 @@ right of the plot to explore:
 
 Follow these steps to recreate this simple line chart:
 
-1. Import the necessary functions from the |bokeh.plotting| module:
+1. Create a new Python file on your machine (e.g. simple_line_chart.py) and open it in a code editor of your choice (such as |Sublime Text|, 
```

Comment:
Links in RST are a bit tricky (see https://docutils.sourceforge.io/docs/user/rst/quickref.html#external-hyperlink-targets).

In this case, I'd use \`Sublime Text`_ in the text and then add the corresponding URL at the bottom of this RST file, i.e. 

```
.. _Sublime Text: https://www.sublimetext.com/
```

Same thing with VS Code: \`Visual Studio Code`_ and then the URL at the bottom of this file:

```
.. _Visual Studio Code: https://code.visualstudio.com/
```